contrib/comfort_models.py

```diff
@@ -361,47 +417,78 @@ def findSaturatedVaporPressureTorr(T):
     HD_S = 1.0 / (RAS + RCLS)
     HE_S = 1.0 / (REAS + RECLS)
 
-    # SET* (standardized humidity, clo, Pb, and CHC)
-    # determined using Newton's iterative solution
-    # FNERRS is defined in the GENERAL SETUP section above
-
     DELTA = .0001
     dx = 100.0
-    X_OLD = TempSkin - HSK / HD_S  # lower bound for SET
+    set_old = round(TempSkin - HSK / HD_S, 2)
     while abs(dx) > .01:
-        ERR1 = (HSK - HD_S * (TempSkin - X_OLD) - W * HE_S
-            * (PSSK - 0.5 * findSaturatedVaporPressureTorr(X_OLD)))
-        ERR2 = (HSK - HD_S * (TempSkin - (X_OLD + DELTA)) - W * HE_S
-            * (PSSK - 0.5 * findSaturatedVaporPressureTorr((X_OLD + DELTA))))
-        X = X_OLD - DELTA * ERR1 / (ERR2 - ERR1)
-        dx = X - X_OLD
-        X_OLD = X
-
-    return X
-
+        ERR1 = (HSK - HD_S * (TempSkin - set_old) - W * HE_S * (PSSK - 0.5 * findSaturatedVaporPressureTorr(set_old)))
+        ERR2 = (HSK - HD_S * (TempSkin - (set_old + DELTA)) - W * HE_S * (PSSK - 0.5 * findSaturatedVaporPressureTorr((set_old + DELTA))))
+        set = set_old - DELTA * ERR1 / (ERR2 - ERR1)
+        dx = set - set_old
+        set_old = set
+    return set
+
+
+def comfAdaptiveComfortASH55(ta, tr, runningMean, vel, eighty_accept_limit_eighty=True, levelOfConditioning=0):
```

Comment:
Should this just be `accept_limit_eighty`?

Reply:
sorry was a typo